survey-runner.tf

```diff
@@ -95,6 +95,7 @@ module "survey-runner-queue" {
   dns_zone_name = "${var.dns_zone_name}"
   aws_key_pair = "${var.aws_key_pair}"
   internet_gateway_id = "${module.survey-runner-vpc.internet_gateway_id}"
+  ebs_snapshot_retention_days = "${var.queue_ebs_snapshot_retention_days}"
```

Comment:
In the latest commit the `queue_ebs_snapshot_retention_days` variable has been renamed `rabbitmq_ebs_snapshot_retention_days` so needs to change here